CMakeLists.txt

```diff
@@ -113,9 +113,9 @@ endif()
 
 
 # WI benckmark =========================================================
-if (BUILD_BENCHMARK)
+#if (BUILD_BENCHMARK)
```

Comment:
If this isn't needed anymore, just remove it. There's no need to have commented-out code cluttering the files.